.github/actions/pnpm-cache/action.yml

```diff
@@ -19,33 +19,35 @@ runs:
     # But this is not available for non-depot runners.
     # So we default to github, and expect the workflow to set cache-provider=buildjet if needed.
     # https://depot.dev/docs/github-actions/overview#faster-caching
+    - name: Setup pnpm
+      uses: pnpm/action-setup@v4
+
     - name: Determine Cache Provider
       id: determine-cache
       shell: bash
       run: echo "cache-provider=${{ inputs.cache-provider }}" >> $GITHUB_ENV
 
+    - name: Get pnpm store directory
+      id: pnpm-store
+      shell: bash
+      run: echo "path=$(pnpm store path)" >> $GITHUB_OUTPUT
```

Comment:
A bit dangerous to call a variable `path`. Random things could go wrong. `store_path` maybe?